docs/website/docs/plus/features/project/source-configuration.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Configuration of Source
+description: Initialize ore sources in yaml file
+---
+
+# Configuration of Sources
+
+<img src="https://storage.googleapis.com/dlt-blog-images/plus/dlt_plus_projects.png" width="500"/>
+
+
+The `dlt.yml` file is used to declaratively configure data sources for use with the dlt CLI. This setup provides a flexible way to manage data extraction for the core sources REST APIs, SQL databases and file systems using YAML syntax.
+
+Credential placeholders for the defined sources are automatically generated in `.dlt/secrets.toml`. Alternatively, credentials may also be provided directly within `dlt.yml`.
+
+
+## REST API
+
+The built-in `rest_api` type enables configuration of REST-based integrations. Multiple endpoints can be defined under a single source.
```

Comment:
Let's add the link to res api source documentation, where people can find details on all available configuration https://deploy-preview-2925--dlt-hub-docs.netlify.app/docs/devel/dlt-ecosystem/verified-sources/rest_api/basic